test/test_autograd.py

```diff
@@ -6856,13 +6861,18 @@ def f(x, y):
 
         x = torch.randn(3)
         y = torch.randn(3)
-        self._check_jacobian_vectorize_correctness(f, (x, y))
+        # The Jacobin computed using forward AD has the dtype of the output
```

Comment:
Is that a problem? Should we document that in the jacobian function?

Reply:
Not sure if it is a problem. But yeah, it's probably good to document.